bootstrap

```diff
@@ -4,6 +4,13 @@
 # Any flags will be passed on to CMake, e.g.,
 # ./bootstrap -DCMAKE_BUILD_TYPE="Debug"
 
+## Check to make cmake is installed, if not offer to install if apt is
```

Comment:
Please fix this comment, i.e. `if not, exit` or similar. Otherwise LGTM, thank you!